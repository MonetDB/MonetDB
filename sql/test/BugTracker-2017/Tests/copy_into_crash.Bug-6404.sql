CREATE TABLE t (i1 int, i2 int, s1 string, PRIMARY KEY (i1));

COPY 1 RECORDS INTO t FROM STDIN (i1, s1);
1|abc

select * from t;

DROP table t;
