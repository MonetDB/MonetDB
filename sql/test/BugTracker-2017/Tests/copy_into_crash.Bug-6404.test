statement ok
CREATE TABLE t (i1 int, i2 int, s1 string, PRIMARY KEY (i1))

statement ok
COPY 1 RECORDS INTO t FROM STDIN (i1, s1)
<COPY_INTO_DATA>
1|abc

query IIT rowsort
select * from t
----
1
NULL
abc

statement ok
DROP table t

