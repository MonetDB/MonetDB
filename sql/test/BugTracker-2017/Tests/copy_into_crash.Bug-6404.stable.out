stdout of test 'copy_into_crash.Bug-6404` in directory 'sql/test/BugTracker-2017` itself:


# 11:27:50 >  
# 11:27:50 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=35557" "--set" "mapi_usock=/var/tmp/mtest-4174/.s.monetdb.35557" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/rc-monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 11:27:50 >  

# MonetDB 5 server v11.27.6
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.330 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:35557/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-4174/.s.monetdb.35557
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded


# 11:27:50 >  
# 11:27:50 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-4174" "--port=35557"
# 11:27:50 >  

#CREATE TABLE t (i1 int, i2 int, s1 string, PRIMARY KEY (i1));
#COPY 1 RECORDS INTO t FROM STDIN (i1, s1);
#1|abc
[ 1	]
#select * from t;
% sys.t,	sys.t,	sys.t # table_name
% i1,	i2,	s1 # name
% int,	int,	clob # type
% 1,	1,	3 # length
[ 1,	NULL,	"abc"	]
#DROP table t;

# 11:27:50 >  
# 11:27:50 >  "Done."
# 11:27:50 >  

